application/dash_application/views/preqc_wgs.py

```diff
@@ -67,26 +72,22 @@
     special_cols["On-target Reads"],
     special_cols["Purity"]
 ]
-most_bamqc_cols = [*BAMQC_COL.values()]
-most_bamqc_cols.remove(BAMQC_COL.BamFile)
 later_col_set = [
     PINERY_COL.PrepKit, PINERY_COL.TissuePreparation,
     PINERY_COL.LibrarySourceTemplateType, PINERY_COL.ExternalName,
     PINERY_COL.GroupID, PINERY_COL.TissueOrigin, PINERY_COL.TissueType,
     PINERY_COL.Institute, INSTRUMENT_COLS.ModelName
 ]
-wgs_table_columns = [*first_col_set, *most_bamqc_cols, *later_col_set]
+wgs_table_columns = [*first_col_set, *BAMQC_COL.values(), *ICHOR_COL.values(), *later_col_set]
```

Comment:
Can the change to remove `most_bamqc_cols` be reverted? I don't think there's any value in the user seeing the path to the BAM file.

Reply:
That column is not in the BamQC3 format.

Reply:
Even better.